src/render-summary.js

```diff
@@ -27,7 +27,7 @@ export default class Summary {
 
         updateUi.call(this);
 
-        window.addEventListener('update', updateUi);
+        window.addEventListener('update', () => updateUi.call(this));
```

Comment:
Looks like you need to rebase on master.

Reply:
yup